node/src/blockchain/blockchain_interface/blockchain_interface_web3/mod.rs

```diff
@@ -463,6 +539,24 @@ mod tests {
                 ]
             }
         )
+
+        // TODO: GH-543: Improve MBCS so we can confirm the calls we make are the correct ones.
+        //       Example of older code
+        //       let requests = test_server.requests_so_far();
+        //         let bodies: Vec<String> = requests
+        //             .into_iter()
+        //             .map(|request| serde_json::from_slice(&request.body()).unwrap())
+        //             .map(|b: Value| serde_json::to_string(&b).unwrap())
+        //             .collect();
+        //         let expected_body_prefix = r#"[{"id":0,"jsonrpc":"2.0","method":"eth_blockNumber","params":[]},{"id":1,"jsonrpc":"2.0","method":"eth_getLogs","params":[{"address":"0x384dec25e03f94931767ce4c3556168468ba24c3","fromBlock":"0x2a","toBlock":"0x400","topics":["0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef",null,"0x000000000000000000000000"#;
+        //         let expected_body_suffix = r#""]}]}]"#;
+        //         let expected_body = format!(
+        //             "{}{}{}",
+        //             expected_body_prefix,
+        //             &to[2..],
+        //             expected_body_suffix
+        //         );
+        //         assert_eq!(bodies, vec!(expected_body));
```

Comment:
I just want to assure myself that you really agree this has been an intentionally left comment.

Reply:
We needed this for historical point of view. So, I have migrated it to the card itself.